Align protocol docs with style guide (#575)
LiskHQ/lisk-docs: modules/ROOT/pages/blocks.adoc

= Blocks

> :description: The blocks section of the Lisk protocol describes the actual data stored in a block and how new blocks are created.

Reviewer: ```suggestion
:description: This section describes the structure and properties of blocks in the Lisk protocol.
```

=== `height`
This property is 1 for the genesis block.
For every other blocks, it is the height of the parent block plus one.

=== `transactionRoot`
The Merkle root of the payload.

=== `generatorPublicKey`
The public key of the delegate who generated the block.

=== `reward`
The reward received by the delegate who generated the block.
The value of the reward is described in the section <<rewards_and_fees, Rewards and fees>>.

=== `signature`
The signature of the block.
This signature has to be generated by the private key associated to `generatorPublicKey`.
This signature scheme is described in the xref:{url_appendix_signature_scheme}[Appendix].

=== `asset`
This property is used to store information regarding the chain that is not necessarily present on all chains created with the SDK.
By default, the block asset contains the three properties below:

* `maxHeightPreviouslyForged`: The maximum height at which the public key in `generatorPublicKey` forged a block.
Please see xref:{url_consensus_delegate_fork_choice_rule}[Consensus votes on blocks and fork choice rule] for more details.

* `maxHeightPrevoted`: The maximum height on the chain which has received 68 prevotes.

> Please see xref:{url_consensus_delegate_fork_choice_rule}[Consensus votes on blocks and fork choice rule], Consensus votes on blocks and fork choice rule>> for more details.

Reviewer: I think `, Consensus votes on blocks and fork choice rule>> ` needs to be removed.
Otherwise, same comment as in transaction file: This style of referencing seems odd.

> Please see xref:{url_consensus_delegate_fork_choice_rule}[Consensus votes on blocks and fork choice rule], Consensus votes on blocks and fork choice rule>> for more details.

Author: ```suggestion
Please see xref:{url_consensus_delegate_fork_choice_rule}[Consensus votes on blocks and fork choice rule] for more details.
```
